CONTRIBUTING.md

```diff
@@ -12,7 +12,7 @@ Follow the [botkube-docs/CONTRIBUTING.md](https://github.com/kubeshop/botkube-do
 
```

Comment:
I think we're still missing this AC:

> describe issue with the botkube-system config map. It must be deleted before executing test for the second time on the same cluster.


See Mateusz's comment: https://botkube.slack.com/archives/CG7TTBXM2/p1665497211898899?thread_ts=1665483992.804649&cid=CG7TTBXM2

Reply:
Will handle.

Reply:
Done.